_includes/release-notes/overview.md

```diff
@@ -8,7 +8,7 @@ TinyMCE 5.0 has an ARIA [WCAG 2.1 standards](https://www.w3.org/WAI/standards-gu
 Include this line of code in the <head> of the HTML page and link to the tinymce.min.js source file:
 
 ```js
-<script src="https://cloud.tinymce.com/5-stable/tinymce.min.js?apiKey=your_API_key"></script>
+<script src="https://cloud.tinymce.com/5/tinymce.min.js?apiKey=your_API_key"></script>
```

Comment:
for all these links
src="https://cloud.tinymce.com/5/tinymce.min.js?apiKey=your_API_key"

replace with 
src="{{ site.cdnurl }}?apiKey=your_API_key"

for maintainability

Reply:
also update the variable {{ site.cdnurl }} to be:

https://cloud.tinymce.com/5/tinymce.min.js